### ISSUE TITLE
Link Original Documentation

### ISSUE THREAD

**Issue by sebastianriese** [fsfw-dresden/hello-world-collection#1]
If we do this, we should include links to the usual introductory documentation.

---

In the case of Go, I would remark, that the official documentation has *very* good introductory content. We can not even come close to compete with it. (Similar points could be said about Python). 

Note, that it would be very desirable for the Go-introduction to explain how to set-up a GOPATH, this in turn might change in future version (though this is unlikely), this means we effectively have to copy the introductory tour of Go, which explains everything nicely, lets you interactively change and try go-code, etc. – so all in all I am against including Go (if not against the whole project). We just make ourselves ridiculous by trying to giving a simpler start to Go than the Go people.

If we try this, we should explain the things that people might actually get wrong (i.e. add a lucid explanation of go path, go get). Additional content should concern the io-system of Go which is quite complex due to several layers of abstraction (with differing limitations).

What I want to say is: If we include Go, we should only include the tricky parts that are not explained or only hinted at in the documentation (and those are few).

**Comment by cknoll** [comment]
I fully agree, that we cannot and should not compete with the official docs. However I think for the [Uni-Stick-Project](https://fsfw-dresden.de/uni-stick) there is some value in having some examples that **run out of the box**. (Apart from delivering these examples in a configured environment their value is indeed questionable but that is true for many other repos too).

**Comment by sebastianriese** [comment]
Oh, I didn't understand the scope the example repository was intended for (the Uni Stick). I think the tag line of this repository should document that scope. (I see the README does so already).
